### PR TITLE
[RISCV] Update Zicntr and Zihpm to version 2p0

### PR DESCRIPTION
**Pull request by jacquesguan**

### CODE REVIEW

**Comment by eopXD** [comment]
Your commit bumps both the counter and timer extension, which makes your commit message inaccurate.

**Comment by jacquesguan** [comment]
> Your commit bumps both the counter and timer extension, which makes your commit message inaccurate.

I think Counters in RISCV ISA include both "Zicntr" and "Zihpm", https://github.com/riscv/riscv-isa-manual/blob/main/src/counters.adoc.

**Comment by eopXD** [comment]
Sorry, that was a mis-statement. There are two counter extensions you are bumping.

Anyway, rewording the commit message is a nit from me.

**Comment by jacquesguan** [comment]
> Sorry, that was a mis-statement. There are two counter extensions you are bumping.
> 
> Anyway, rewording the commit message is a nit from me.

OK, I retitle the commit.

File: llvm/unittests/Support/RISCVISAInfoTest.cpp [hunk 1]
```diff
@@ -629,7 +629,7 @@ TEST(getTargetFeatureForExtension, RetrieveTargetFeatureFromOneExt) {
 
 TEST(RiscvExtensionsHelp, CheckExtensions) {
   std::string ExpectedOutput =
-R"(All available -march extensions for RISC-V
+      R"(All available -march extensions for RISC-V
```
Comment (eopXD) [inline llvm/unittests/Support/RISCVISAInfoTest.cpp:632]: This change doesn't seem necessary

Comment (jacquesguan) [reply llvm/unittests/Support/RISCVISAInfoTest.cpp]: A wrong format, fixed now.

**Review by eopXD — APPROVED** [review]

**Pull request merged**